src/components/FromTitle.tsx

```diff
@@ -15,9 +15,9 @@ export const FromTitle = ({
   isHoveringProfile,
   isReply,
 }: FromTitleProps): JSX.Element => {
-  const atHandle = userInfo.handle && "@" + userInfo.handle;
+  const atHandle = userInfo?.handle && "@" + userInfo?.handle;
```

Comment:
issues with testing so had to add the "?"s in

Reply:
That would imply that our prop type is wrong. userInfo should exist, but handle might not. I wonder if we can fix the tests to take these optionals back out.